src/docs/product/performance/filters-display/index.mdx

```diff
@@ -38,6 +38,23 @@ When you select "All Transactions", the table includes all transactions across a
 
 "Mobile" puts an emphasis on [Mobile Vitals](/product/performance/mobile-vitals/). Note that if you've selected a non-mobile project, this mode also displays all of those transactions.
 
+## Transaction Search
```

Comment:
This section feels like it fits nicely under the heading just below! You could rename "Search Condition Filters" to "Transaction Search", and combine them. Both sections are about the search bar, and this `<Note>` could be a caveat